src/engine/source/builder/helperParser.hpp

```diff
@@ -402,7 +503,43 @@ inline parsec::Parser<ExpressionToken> getExpressionParser()
         return parsec::makeSuccess(std::move(word), next);
     };
 
-    auto valueParser = valueRefParser | jsonParser | wordParser;
+    parsec::Parser<json::Json> singleQuotesParser = [](auto sv, auto pos) -> parsec::Result<json::Json>
+    {
+        using namespace base::utils::string;
+        auto next = pos;
+        // Check if start with single quote
+        if (next + 1 >= sv.size() || sv[next] != syntax::SINGLE_QUOTE)
+        {
+            return parsec::makeError<json::Json>("Single quote expected", pos);
+        }
+        next++;
+
+        // Find end of string (unescaped single quote)
+        bool valid = false;
+        for (; next < sv.size(); ++next)
+        {
+            if (sv[next] == syntax::SINGLE_QUOTE && sv[next - 1] != syntax::FUNCTION_HELPER_DEFAULT_ESCAPE)
+            {
+                valid = true;
+                break;
+            }
+        }
+
+        if (!valid)
+        {
+            return parsec::makeError<json::Json>("Invalid single quote string", pos);
+        }
+
+        const auto quoted = sv.substr(pos + 1, next - pos - 1);
+        const auto str = unescapeString(quoted, syntax::FUNCTION_HELPER_DEFAULT_ESCAPE, syntax::SINGLE_QUOTE, false);
+
+        json::Json value;
```

Comment:
nitpick: initialize variable

Reply:
It is not a native type, so it follows the initialization rules, using its default parameterless constructor:
https://en.cppreference.com/w/cpp/language/data_members#Member_initialization
Immediately after creating it with its default value (json null) it is replaced by a "json string"